go.sh

```diff
@@ -0,0 +1,6 @@
+export NODE_ENV=development
+echo $NODE_ENV
+rm -rf ./dist/*
+yarn build
+cp -Rf ./dist/* $MY_DPC_CLIENT
+
```

Comment:
What is this go.sh file for?

Reply:
it might not work for the client. As its packaged for NPM

Reply:
it is for local testing, for users, who want to use directly without using npm

Reply:
removed